src/measure.js

```diff
@@ -0,0 +1,26 @@
+goog.module('measurementlibrary.measure');
+
+const {DataLayerHelper} = goog.require('helper');
+
+/**
+ * Listen to events passed to the current window.dataLayer (or a new one
+ * if none exist).
+ */
+function setup() {
+  /**
+   * @global {!Array} The data layer of the application, storing a record of
+   * events that have changed over time.
+   */
+  window.dataLayer = window.dataLayer || [];
```

Comment:
Does the dataLayer always need to be named "dataLayer"? If not, maybe add it as a parameter to "setup" and pass it in?

Reply:
I think it will almost always be named dataLayer, but maybe there is another use case I haven't thought of so I will add the default parameter